ee/fleetd-chrome/src/tables/privacy_preferences.ts

```diff
@@ -71,13 +71,13 @@ export default class TablePrivacyPreferences extends Table {
             } else {
               propertyAPI.get({}, (details: ChromeSettingGetResultDetails) => {
                 if (property === "web_rtc_ip_handling_policy") {
-                  resolve({[property]: details.value});
+                  resolve({ [property]: details.value });
                 } else {
-                  // convert bool response to string binary flag
+                  // convert bool response to binary flag
                   if (details.value === true) {
-                    resolve({ [property]: "1" });
+                    resolve({ [property]: 1 });
```

Comment:
@getvictor `String(true)` gives us `"true"`, whereas this currently gives us "1" and "0" (after conversion) – would this break the API?

Reply:
Yes, "true" instead of "1" would break the API. SQLite does not have a boolean.

This logic converts true to "1" and false to "0" -- that should always be the case for all tables. So we should put this logic in the common place.